cosim/black-parrot-example/verilator/Makefile

```diff
@@ -0,0 +1,72 @@
+
+## Set environment variables
+TOPLEVEL ?= $(shell git rev-parse --show-toplevel)
+
+export BLACKPARROT_DIR := $(abspath ../../import/black-parrot)
+export BP_SDK_DIR      := $(abspath ../../import/sdk)
+export BSG_ZYNQ_PL_SHELL_DIR:= $(abspath ../../common/v)
+export BP_FE_DIR        = $(BLACKPARROT_DIR)/bp_fe
+export BP_COMMON_DIR    = $(BLACKPARROT_DIR)/bp_common
+export BP_BE_DIR        = $(BLACKPARROT_DIR)/bp_be
+export BP_ME_DIR        = $(BLACKPARROT_DIR)/bp_me
+export BP_TOP_DIR       = $(BLACKPARROT_DIR)/bp_top
+export BP_EXTERNAL_DIR  = $(BLACKPARROT_DIR)/external
+export BASEJUMP_STL_DIR = $(BP_EXTERNAL_DIR)/basejump_stl
+export HARDFLOAT_DIR    = $(BP_EXTERNAL_DIR)/HardFloat
+export BP_TOOLS_DIR     = $(BLACKPARROT_DIR)/tools
+
+export PATH := $(BP_TOOLS_DIR)/install/bin:$(PATH)
+
+export TEST_DIR ?= .
+export TOP  ?= top
+export HOST-PROGRAM ?= ps.cpp
+NBF_FILE=../prog.nbf
+
+$(warning Using BSG_ZYNQ_PL_SHELL_DIR=$(BSG_ZYNQ_PL_SHELL_DIR))
+$(warning Using BLACKPARROT_DIR=$(BLACKPARROT_DIR))
+
```

Comment:
We should check for the correct verilator version (4+?).  Default behavior on ubuntu/centos is to use an old version which fails with a "option --build not found" error

Reply:
added fixme comment. =)